tests/models/transformer_builder_tests.py

```diff
@@ -1,31 +0,0 @@
-import unittest
```

Comment:
Why are we removing this test?

Reply:
See other comment

Reply:
The model builder fails the tests too? HFTransformerBuilder is being used in gen_external_params.py so still has a use for being tested (reminds me I need to make tests for that still)